### PR TITLE
Bug 1827486: clarify usage of 'oc adm create-kubeconfig'

### PR DESCRIPTION
**Pull request by sallyom**
As discussed on slack, we will deprecate 'oc adm create-kubeconfig' in its current form. Will replace with an equivalent 'oc config set ...' in the future.  

/assign @soltysh

### CODE REVIEW

File: pkg/cli/admin/createkubeconfig/create_kubeconfig.go [hunk 1]
```diff
@@ -93,6 +93,8 @@ func NewCommandCreateKubeConfig(commandName string, fullName string, streams gen
 				kcmdutil.CheckErr(err)
 			}
 		},
+		Deprecated: "and will be removed in the future version",
+		Hidden:     true,
```
Comment (soltysh) [inline pkg/cli/admin/createkubeconfig/create_kubeconfig.go:97]: Nit: this is not needed, each deprecated command is automatically hidden.

Comment (sallyom) [reply pkg/cli/admin/createkubeconfig/create_kubeconfig.go]: oops and updated

**Pull request merged**